Ical.Net/Evaluation/RecurrencePatternEvaluator.cs

```diff
@@ -222,6 +222,18 @@ private IEnumerable<DateTime> EnumerateDates(DateTime originalDate, DateTime see
         }
     }
 
+    private sealed class ExpandContext
```

Comment:
use a `struct` or `struct record` instead, or even a `bool` instead?

Reply:
Yes, could be, and pass it by ref. Thought about it but went for simplicity. The overhead is rather low, but agree, struct would be nicer nevertheless. Will check.